src/agent/windows/WMICommunication.cpp

```diff
@@ -207,9 +209,23 @@ bool WMICommunication::GetSMARTDataViaWMI()
 
 void WMICommunication::FeedSmartDataStructure(std::vector<BYTE>& data, const LONG& dataSize)
 {
-    for (int i = 0; i < dataSize; ++i)
+
+    for (int iterator = 0; data.size() > iterator; iterator += 12)
```

Comment:
iterator? what is wrong with 'i' ;)

Reply:
What is wrong with interator? :>

Reply:
It is **very** common to use i. Maybe something like 'pos' or even 'position' is better here? But 'iterator' looks way to broad